src/main/fc/runtime_config.c

```diff
@@ -29,6 +32,37 @@ uint32_t flightModeFlags = 0;
 
 static uint32_t enabledSensors = 0;
 
+// XXX: This function assumes that all flags in ARMING_DISABLED_ALL_FLAGS
```

Comment:
Why? You can simply iterate through set bits of `ARMING_DISABLED_ALL_FLAGS` and check if corresponding bit it set in `armingFlags`.  You are returning the bitmask anyway.

Reply:
I tried to save a few cycles by checking just the relevant bits, but I will change that. Performance doesn't matter in this function since it will never be called while the aircraft is armed.